src/common/helpers/start-server.js

```diff
@@ -11,7 +11,7 @@ export async function startServer() {
     `Access your backend on http://localhost:${config.get('port')}`
   )
 
-  await startMessagingService(server.logger)
+  await startMessagingService(server)
```

Comment:
not sure this change is needed/warranted. Looks like you passed it in to get access to the DB, but then aren't using it anyway. Maybe just restore it to what it was?